src/main/resources/META-INF/persistence.xml

```diff
@@ -8,6 +8,8 @@
 		<provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
 		<jta-data-source>jdbc/car-app</jta-data-source>	
 		<class>com.car.app.model.Car</class>
+		<class>com.car.app.model.Country</class>
```

Comment:
Adding the classes manually is a bit of a pain, because every time a new entity is added you must manually add it here. What if we forget to add it here ? Let's do the framework do this work for us. If we do not add the classes here the framework will automatically pick them all.